[2025-10-21T01:42:54.072+0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-21T01:42:58.033+0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_etl.load_task manual__2025-10-20T22:31:28.253460+00:00 [queued]>
[2025-10-21T01:42:59.609+0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_etl.load_task manual__2025-10-20T22:31:28.253460+00:00 [queued]>
[2025-10-21T01:42:59.611+0300] {taskinstance.py:2306} INFO - Starting attempt 2 of 4
[2025-10-21T01:43:01.605+0300] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_task> on 2025-10-20 22:31:28.253460+00:00
[2025-10-21T01:43:01.647+0300] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'stock_etl', 'load_task', 'manual__2025-10-20T22:31:28.253460+00:00', '--job-id', '125', '--raw', '--subdir', 'DAGS_FOLDER/stock_etl_dag.py', '--cfg-path', '/tmp/tmp94dqo2jf']
[2025-10-21T01:43:01.662+0300] {standard_task_runner.py:91} INFO - Job 125: Subtask load_task
[2025-10-21T01:43:01.670+0300] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=331) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-21T01:43:01.675+0300] {standard_task_runner.py:64} INFO - Started process 343 to run task
[2025-10-21T01:43:11.908+0300] {local_task_job_runner.py:124} ERROR - Received SIGTERM. Terminating subprocesses
[2025-10-21T01:43:11.920+0300] {process_utils.py:132} INFO - Sending 15 to group 343. PIDs of all processes in the group: [343]
[2025-10-21T01:43:11.922+0300] {process_utils.py:87} INFO - Sending the signal 15 to group 343
[2025-10-21T01:43:11.956+0300] {process_utils.py:80} INFO - Process psutil.Process(pid=343, status='terminated', exitcode=<Negsignal.SIGTERM: -15>, started='01:43:00') (343) terminated with exit code -15
[2025-10-21T01:43:11.962+0300] {local_task_job_runner.py:243} INFO - Task exited with return code 143
[2025-10-21T01:43:18.434+0300] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-21T01:43:18.741+0300] {local_task_job_runner.py:222} INFO - ::endgroup::
